requirements/requirements-testing.txt

```diff
@@ -5,3 +5,4 @@ pytest-django>=4.5.2,<5.0
 importlib-metadata<5.0
 # temporary pin of attrs
 attrs==22.1.0
+pytz
```

Comment:
I'm not sure about that one... Would it risk hiding some problem when `pytz` is NOT installed? 

Ideally, I would like to have:

- `pytz` not explicitly installed by default
- Have a couple/few tox envs testing with `pytz` installed

Not sure if that would make the matrix too complex

Reply:
> Would it risk hiding some problem when pytz is NOT installed?

Which kind of issues?

Reply:
Well, I'm talking about an unknown unknown, so I don't have a concrete example, I'm merely asking the question 😄

Reply:
I guess the matrix can be extended to run tests against Django 4 once with pytz and again without it (that would also mean that the second skip has to return :)) but I wonder whether that's necessary.